playwright-e2e/pages/start-application.ts

```diff
@@ -53,17 +54,14 @@ export class StartApplication {
     this.childDetailsUpdated = page.locator('p').filter({ hasText: 'Child\'s Details' }).getByRole('img', { name: 'Information added' });
     this.welshLanguageRequirements = page.getByRole('link', { name: 'Welsh language requirements' });
     this.welshLanguageReqFinished = page.locator('p:has(a[text="Welsh language requirements"]) > img[title="Finished"]');
-    
     this.internationalElementsHeading = page.getByRole('link', { name: 'International element' });
-    this.submitApplicationLink = page.getByRole('link', { name: 'Submit application' });
-
-
-    this.courtServicesNeeded = page.getByRole('link', { name: 'Court services needed' });
+    this.submitApplicationLink = page.getByRole('link', { name: 'Submit application' })
+    this.otherProceedingsLink = page.getByRole("link", { name: "Other Proceedings", });
+    this.courtServicesNeeded = page.getByRole('link', { name: 'Court services needed' 
```

Comment:
missing }); on this line - causes the test errors on the pipeline
```suggestion
    this.courtServicesNeeded = page.getByRole('link', { name: 'Court services needed' });
```